apps/discovery_api/test/integration/docker-compose.yaml

```diff
@@ -1,67 +1,80 @@
-version: "3"
+---
+version: '3.4'
 services:
-  redis:
-    image: redis
-    ports:
-      - "6379:6379"
-    healthcheck:
-      test: ["CMD", "redis-cli", "PING"]
-      interval: 5s
-      timeout: 20s
-      retries: 3
-  presto:
-    image: smartcitiesdata/presto-testo:0.9.12
-    depends_on:
-      - metastore
-      - minio
-    ports:
-      - "8080:8080"
-    healthcheck:
-      test:
-        [
-          "CMD-SHELL",
-          'curl -s http://localhost:8080/v1/info | grep -q ''"starting":false''',
-        ]
-      interval: 10s
-      timeout: 30s
-      retries: 10
+  init:
+    image: alpine:3.16.0
+    volumes:
+      - ${PWD}/test/integration:/workdir
+      - minio:/minio
+    entrypoint: /workdir/setup.sh
   metastore:
-    image: smartcitiesdata/metastore-testo:0.9.12
+    image: quay.io/cloudservices/ubi-hive:3.1.2-metastore-008
     depends_on:
-      - postgres
+    - postgres
+    - minio
     ports:
-      - "9083:9083"
-    healthcheck:
-      test: ["CMD-SHELL", "nc -z localhost 9083"]
-      interval: 10s
-      timeout: 30s
-      retries: 3
-    command: >
-      /bin/bash -c "
-        /opt/hive-metastore/bin/schematool -dbType postgres -validate || /opt/hive-metastore/bin/schematool -dbType postgres -initSchema;
-        /opt/hive-metastore/bin/start-metastore"
+    - "8000:8000"
+    environment:
+      HOSTNAME: localhost
+      S3_HOST: minio
+      S3_PORT: 9000
+      S3_ACCESS_KEY: minioadmin
+      S3_SECRET: minioadmin
+      S3_BUCKET_NAME: trino-hive-storage
+      S3_BUCKET_PATH: hive-s3
+      DATABASE_USER: hive
+      DATABASE_PASSWORD: password123
+      POSTGRES_SQL_SERVICE_HOST: postgres
+      POSTGRES_SQL_SERVICE_PORT: 5432
+      DATABASE_NAME: metastore
+
   postgres:
-    image: smartcitiesdata/postgres-testo:0.9.12
+    image: postgres:14.4-alpine
+    environment:
+      POSTGRES_USER: hive
+      POSTGRES_PASSWORD: password123
+      POSTGRES_DB: metastore
     ports:
       - "5455:5432"
   ecto-postgres:
-    image: postgres:9.6.16
+    image: postgres:14.4-alpine
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: discovery_api_test
     ports:
       - "5456:5432"
-    volumes:
-      - "./support/init_postgres.sql:/docker-entrypoint-initdb.d/init_postgres.sql"
   minio:
-    image: smartcitiesdata/minio-testo:0.9.12
+    image: minio/minio:RELEASE.2022-07-17T15-43-14Z.fips
+    command: server /data --console-address ":9001"
+    volumes:
+      - minio:/data
+    ports:
+    - "9000:9000"
+    - "9001:9001"
+    depends_on:
+      - init
+  presto:
```

Comment:
are we keeping references to presto in the codebase? Can this image be renamed to trino?

Reply:
There are 200+ mentions of `Presto` in the codebase, including a view "Presto" services, so nevermind this 👍